packages/neos-ui-editors/src/Boolean/index.js

```diff
@@ -47,7 +46,8 @@ BooleanEditor.propTypes = {
     label: PropTypes.string.isRequired,
     value: PropTypes.oneOfType([PropTypes.bool, PropTypes.string]),
     commit: PropTypes.func.isRequired,
-    isDisabled: PropTypes.bool
+    isDisabled: PropTypes.bool,
```

Comment:
This line can be removed as the disabled state comes from the options and isDisabled is also never used.